src/Ninja/G1CP/Content/Tests/test166.d

```diff
@@ -0,0 +1,29 @@
+/*
+ * #166 Ambient dialogs of rogue not available
+ *
+ * There does not seem an easy way to test this fix programmatically, so this test relies on manual confirmation.
```

Comment:
There should actually be a way to check this with an automatic test. As found in similar tests, set the dialog `Info_ORG_829_OfferJoint` to told, trigger the condition function, and check its return value.